CMakeSettings.json

```diff
@@ -33,40 +33,40 @@
 
   "configurations": [
     {
-      "name": "x86-Debug",
+      "name": "x64-Debug",
       "generator": "Ninja",
       "configurationType": "Debug",
-      "inheritEnvironments": [ "msvc_x86" ],
-      "buildRoot": "${env.USERPROFILE}\\CMakeBuilds\\${workspaceHash}\\build\\${name}",
+      "inheritEnvironments": [ "msvc_x64_x64" ],
+      "buildRoot": "${workspaceRoot}\\build\\Debug",
       "installRoot": "${env.USERPROFILE}\\CMakeBuilds\\${workspaceHash}\\install\\${name}",
-      "cmakeCommandArgs": "",
+      "cmakeCommandArgs": "-DBUILD_ENCLAVES=1",
```

Comment:
I actually don't think we should have this file in the repo it all. For one, it's auto-generated, and for two, any changes made are user specific. We should provide instructions that, on Windows, you need to add `-DBUILD_ENCLAVES=ON` to `cmakeCommandArgs` if building this particular way in Visual Studio, but I would suggest deleting `CMakeSettings.json` and adding it to `.gitignore`.

Reply:
We could even go one further and provide a sample configuration suitable for copying into a user's `CMakeSettings.json`.

Reply:
I think this is worth keeping around, in particular for its removal of the x86 (unsupported) configuration which VS defaults to, and its choice of `buildRoot` as a bit of automatic configuration so that we can later set up debugging to a known target.

Reply:
Fair enough.